### PR TITLE
New Docs Site deployment structure info

### PR DESCRIPTION
**Pull request by daremiyo**

### CODE REVIEW

File: using-github.md [hunk 1]
```diff
@@ -36,5 +36,6 @@ To work with GitHub:
...
-6. A [README file](/docs-guide/add-content#readme), as specified in this documentation, is included in the template.
-7. A [License](/docs-guide/add-content#license), as specified in this documentation, is included in the template.
+6. Add a [GitHub Webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks) to the repository. Webhooks are added in the Settings page of a repository. The Webhook listens for `push` events which push the updates to the development and production servers, and it should point to the AWS API Gateway endpoint - build_docs_site_api. Contact the Head of Digital Strategies for the GH_TOKEN as part of the webhook configuration. For more information, see the [Documentation Site Info Sheet](https://docs.rockarch.org/systems-info-sheets/documentation-site-info-sheet/).
```
Comment (helrond) [inline using-github.md]:
```suggestion
6. Add a [GitHub Webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks) to the repository. Webhooks are added in the Settings page of a repository. The Webhook listens for `push` events which push the updates to the development and production servers, and it should point to the AWS API Gateway endpoint for the `build_docs_site_api`. Contact the Head of Digital Strategies for the token and AWS Gateway endpoint URL you will need as part of the webhook configuration. For more information, see the [Documentation Site Info Sheet](https://docs.rockarch.org/systems-info-sheets/documentation-site-info-sheet/).
```

**Review by helrond — CHANGES_REQUESTED** [review]
Just one tweak to the language.

File: add-content.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ Prior to adding or editing content, all additions and changes must be approved a
 1. Create a new RAC GitHub repository for the documentation if one does not exist using the [docs-template](https://github.com/RockefellerArchiveCenter/docs-template) repository template. Contact an RAC employee with administrative access to the RAC GitHub to enable this action. See [Create a Repo](/docs-guide/using-github#create-a-repo).
 2. Create a branch called `development` in the target repository.
 3. Commit new content or changes to the `development` branch using short, descriptive commit messages that describe the changes like "add config values," "fix typo," or "change template text". Use this descriptive commit message instead of the default "update [file]" message provided in GitHub.
-4. Review the changes on the RAC development server at docs.dev.rockarch.org. When a repository is first created, it has to be manually added to the development server. See the [Documentation Site Info Sheet](http://docs.rockarch.org/systems-info-sheets/documentation-site-info-sheet) or ask for help.
+4. Review the changes on the RAC development server at docs.dev.rockarch.org.
```
Comment (HaSistrunk) [inline add-content.md]:
```suggestion
4. Review the changes on the [RAC development site](docs.dev.rockarch.org).
```

**Comment by HaSistrunk** [comment]
One step I think we need to add is to add new documentation to the `repositories.yml` file in the `docs-build` repo as specified in the docs-build README: https://github.com/RockefellerArchiveCenter/docs-build#adding-or-removing-repositories

**Comment by daremiyo** [comment]
> One step I think we need to add is to add new documentation to the `repositories.yml` file in the `docs-build` repo as specified in the docs-build README: https://github.com/RockefellerArchiveCenter/docs-build#adding-or-removing-repositories
 
Thanks for the catch!

**Review by helrond — APPROVED** [review]

**Review by kemartin2015 — APPROVED** [review]

**Comment by daremiyo** [comment]
hi @HaSistrunk I put in a step for adding new documentation to the repositories.yml file under the Create a Repo section

**Review by HaSistrunk — APPROVED** [review]
Looks good!

**Pull request merged**